app/src/main/res/values/strings.xml

```diff
@@ -28,52 +28,52 @@
+    <string name="barcodeImageDescription">Image of card barcode</string>
 
-    <string name="noStoreError">No Store entered</string>
-    <string name="noCardIdError">No Card ID entered</string>
-    <string name="noCardExistsError">Could not lookup loyalty card</string>
+    <string name="noStoreError">No store entered</string>
+    <string name="noCardIdError">No card ID entered</string>
+    <string name="noCardExistsError">Could not find card</string>
     <string name="failedParsingImportUriError">Could not parse the import URI</string>
     <string name="cardIdFormat">%1$s: %2$s</string>
     <string name="importExport">Import/Export</string>
     <string name="exportName">Export</string>
-    <string name="importExportHelp">Backed up data can allow you to move your cards to another device.</string>
-    <string name="importSuccessfulTitle">Import successful</string>
+    <string name="importExportHelp">Backing up your cards allows you to move them to another device.</string>
+    <string name="importSuccessfulTitle">Imported</string>
     <string name="importFailedTitle">Import failed</string>
-    <string name="importFailed">Failed to import</string>
-    <string name="exportSuccessfulTitle">Export successful</string>
+    <string name="importFailed">Could not import cards</string>
+    <string name="exportSuccessfulTitle">Exported</string>
     <string name="exportFailedTitle">Export failed</string>
-    <string name="exportFailed">Failed to export</string>
+    <string name="exportFailed">Could not export cards</string>
     <string name="importing">Importing&#8230;</string>
     <string name="exporting">Exporting&#8230;</string>
-    <string name="noExternalStoragePermissionError">Unable to import or export cards without the external storage permission</string>
-    <string name="exportOptionExplanation">Data will be written to a location of your choice.</string>
+    <string name="noExternalStoragePermissionError">Grant external storage permission to import or export cards first</string>
+    <string name="exportOptionExplanation">The data will be written to a location of your choice.</string>
     <string name="importOptionFilesystemTitle">Import from filesystem</string>
     <string name="importOptionFilesystemExplanation">Choose a specific file from the filesystem.</string>
     <string name="importOptionFilesystemButton">From filesystem</string>
-    <string name="importOptionApplicationTitle">Use external application</string>
-    <string name="importOptionApplicationExplanation">Use an external application like Dropbox, Google Drive, or your favorite file manager to open a file.</string>
-    <string name="importOptionApplicationButton">Use external application</string>
+    <string name="importOptionApplicationTitle">Use external app</string>
+    <string name="importOptionApplicationExplanation">Use an external app like Nextcloud, OnionShare, or your favorite file manager to open a file.</string>
```

Comment:
Nextcloud is popular enough to add, but I have never used OnionShare. I think mentioning Dropbox and Google Drive could help to let users understand.

Otherwise, maybe simplify it is "Use an external app or your favorite file manager to open a file."

Reply:
I think your way is better. Was trying to avoid the F-Droid "reccomends non-free something".